packages/server/src/errors.ts

```diff
@@ -113,12 +43,14 @@ function enrichError(error: Partial<GraphQLError>, debug: boolean = false) {
     },
   });
 
+  const originalErrorExtensions =
```

Comment:
Not sure this name makes sense to me — you're not reading `extensions` into it? (Not that I completely understand this function in general.)

Reply:
Fixed, in updated code.